CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+# Contributing to Pipeline Global Library for ci.jenkins.io
+
+👏 First off, thanks a lot for taking the time to contribute to this global library of pipeline components for [ci.jenkins.io](https://ci.jenkins.io/)!
```

Comment:
This library is used by all of our Jenkins instances
* infra.ci.jenkins.io
* release.ci.jenkins.io
* trusted.ci.jenkins.io
* ci.jenkins.io
:)

Reply:
WDYT of the following?

```suggestion
👏 First off, thanks a lot for taking the time to contribute to this global library of pipeline components for [ci.jenkins.io](https://ci.jenkins.io/) (and many more Jenkins controllers of the [Jenkins Infrastructure Platform](https://www.jenkins.io/projects/infrastructure/)!
```